zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/user/IdentitySetupInitializer.java

```diff
@@ -33,12 +36,15 @@ public final class IdentitySetupInitializer implements StreamProcessorLifecycleA
   public static final String DEFAULT_USER_PASSWORD = "demo";
   public static final String DEFAULT_USER_EMAIL = "demo@demo.com";
   public static final String DEFAULT_ROLE_NAME = "Admin";
+  public static final String DEFAULT_TENANT_ID = TenantOwned.DEFAULT_TENANT_IDENTIFIER;
+  public static final String DEFAULT_TENANT_NAME = "Default";
```

Comment:
:thought_balloon: As you can see I've used the full might of my creative powers on this one. Not sure if the name needs to be discussed somewhere.

Reply:
You actually guessed exactly what's used in the old Identity. So it's perfect 😄